test/crdsvalidation/dataplane_test.go

```diff
@@ -60,6 +60,106 @@ func TestDataPlane(t *testing.T) {
 					},
 				},
 			},
+			{
+				Name: "dbmode off is supported",
+				TestObject: &operatorv1beta1.DataPlane{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName: "dp-",
+						Namespace:    ns.Name,
+					},
+					Spec: operatorv1beta1.DataPlaneSpec{
+						DataPlaneOptions: operatorv1beta1.DataPlaneOptions{
+							Deployment: operatorv1beta1.DataPlaneDeploymentOptions{
+								DeploymentOptions: operatorv1beta1.DeploymentOptions{
+									PodTemplateSpec: &corev1.PodTemplateSpec{
+										Spec: corev1.PodSpec{
+											Containers: []corev1.Container{
+												{
+													Name:  "proxy",
+													Image: "kong:3.9",
+													Env: []corev1.EnvVar{
+														{
+															Name:  "KONG_DATABASE",
+															Value: "",
+														},
+													},
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			{
+				Name: "dbmode '' is supported",
```

Comment:
Nit: `dbmode '' is supported` and `dbmode 'off' is supported` test cases' names should be the other way around 🙂

Reply:
🤦 6b95094b29ab74d03519c932cad00729b90ce05e